<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWidget</class>
 <widget class="QWidget" name="LoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#LoginWidget {
	background-image: url(&quot;:img/login.png&quot;);
	background-repeat: no-repeat;
}
QCheckBox {
	color: #392518;
}

QLabel#l_remember_me {
	color: #453327;
}

QLabel#l_story_awaits {
	color: #453327;
}

QLabel#l_sign_up {
	color: #FF441C;
}

QLineEdit {
	color:#392518;
	background-color: transparent;
 	padding: 0px 23px;
	border: none;
	placeholder-text-color: #392518;
}
QLabel {
	color: #4F1717;
}

QPushButton#btn_login {
    border-image: url(:/img/btn/login.png);
    background-repeat: no-repeat;
}

QPushButton#btn_login:hover {
    border-image: url(:/img/btn/login_hover.png);
    background-repeat: no-repeat;
}

QPushButton#btn_login:pressed  {
    border-image: url(:/img/btn/login_pressed.png);
    background-repeat: no-repeat;
}

QPushButton#btn_login:disabled {
	 border-image: url(:/img/btn/login_disabled.png);
    background-repeat: no-repeat;
}

</string>
  </property>
  <widget class="QLabel" name="l_story_awaits">
   <property name="geometry">
    <rect>
     <x>266</x>
     <y>218</y>
     <width>135</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothic</family>
     <pointsize>12</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
     <hintingpreference>PreferNoHinting</hintingpreference>
     <fontweight>DemiBold</fontweight>
    </font>
   </property>
   <property name="text">
    <string>Your Story awaits!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_login">
   <property name="geometry">
    <rect>
     <x>361</x>
     <y>45</y>
     <width>112</width>
     <height>111</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>112</width>
     <height>111</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="l_sign_up">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>215</y>
     <width>65</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>EurostileBla</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
     <underline>true</underline>
     <hintingpreference>PreferNoHinting</hintingpreference>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;https://storyofalicia.com&quot;&gt;&lt;span style=&quot; font-size:18px; color:#FF441C;&quot;&gt;Sign Up&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="openExternalLinks">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_username">
   <property name="geometry">
    <rect>
     <x>43</x>
     <y>45</y>
     <width>304</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NotEurostile</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
     <hintingpreference>PreferNoHinting</hintingpreference>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>USERNAME</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_remember">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>188</y>
     <width>13</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothic</family>
     <pointsize>13</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_password">
   <property name="geometry">
    <rect>
     <x>43</x>
     <y>109</y>
     <width>304</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NotEurostile</family>
     <pointsize>12</pointsize>
     <hintingpreference>PreferNoHinting</hintingpreference>
     <fontweight>Black</fontweight>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
   <property name="placeholderText">
    <string>PASSWORD</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_remember_me">
   <property name="geometry">
    <rect>
     <x>67</x>
     <y>189</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothic</family>
     <pointsize>13</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
     <hintingpreference>PreferNoHinting</hintingpreference>
     <fontweight>DemiBold</fontweight>
    </font>
   </property>
   <property name="text">
    <string>Remember me</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="master.qrc"/>
 </resources>
 <connections/>
</ui>
